dvc/data/db/webdav.py

```diff
@@ -0,0 +1,27 @@
+from typing import TYPE_CHECKING
+
+from dvc.objects.db import ObjectDB
+
+if TYPE_CHECKING:
+    from dvc.fs.base import FileSystem
+
+
+class WebdavObjectDB(ObjectDB):
```

Comment:
We could just set `fs_path=""` in `get_odb` method, but I wanted to make webdav special. Therefore this method exists.